app/views/pages/webhooks.html.md

```diff
@@ -0,0 +1,109 @@
+# Webhook Notifications
+
+## 1. Introduction
+Webhook notifications allow suppliers to subscribe to the PECS API system and receive notifications of when a move record is created or updated. The notification is transmitted via an HTTP POST to an endpoint specified by the supplier. The notification is signed with a shared key to ensure the authenticity of the message. If the endpoint is unavailable then the PECS system will re-attempt the delivery later, retrying up to 25 times.
+
+
+## 2. Subscribing to notifications
+The subscription process is currently manual - please contact the Book a Secure Move via your Slack channel with the following details:
+
+* Supplier name
```

Comment:
Suppliers have individual Slack channels, so we don't need them to specify their name. It might be worth us getting them to specify an "application name" though, as they could have multiple systems receiving notifications.